WebGUI/js/Desktop.js

```diff
@@ -2473,6 +2473,27 @@ Desktop.handleWindowHelp = function (mouseEvent)
 	return false;
 } //end handleWindowHelp()
 
+//==============================================================================
+Desktop.handleDashboardHelp = function (mouseEvent) 
+{
+	// Check if window exists 
+	if(Desktop.desktop.getForeWindow() == 0)
+	{
+		window.open("https://github.com/art-daq", "_blank")
```

Comment:
Please make the default ? link this: https://github.com/art-daq/otsdaq#readme

Reply:
Updated.
https://github.com/art-daq/otsdaq_utilities/pull/38/commits/7462e3442777cf9c883e08c18d6df236f8abbd50